tests/acquia_lift.test

```diff
@@ -473,27 +481,35 @@ class AcquiaLiftWebTestConfig extends AcquiaLiftWebTestBase {
     );
   }
 
-  public function testLiftAgentAvailibility() {
-    // remove the account information
-    $lift_info = variable_get('acquia_lift_account_info', array());
+  public function testLiftAgentAvailability() {
+    // Fill all required information.
+    $this->configureAcquiaLiftAccount();
+    // Remove the account information.
     variable_del('acquia_lift_account_info');
```

Comment:
Haha I was trying to get the test to say "if we have the full configuration but missing this _particular_ variable, then you'll see this _particular_ error message".